src/components/MultiLink/MultiLink.vue

```diff
@@ -36,9 +36,10 @@ export default Vue.extend({
     simulationLinks(): SimulationLink[] | null {
       if (this.network !== null) {
         return this.network.edges.map((link: Link) => {
-          link.source = link._from;
-          link.target = link._to;
-          return (link as SimulationLink);
+          const newLink = link;
+          newLink.source = link._from;
+          newLink.target = link._to;
+          return (newLink as SimulationLink);
```

Comment:
Instead of using `as`, I'd recommend typing `newLink` on line 39 directly.

While you're at it, I'd say you should define `newLink` in one go as an object literal rather than declaring it and using assignment on its fields (you could do this with the spread operator). This is mostly a style consideration, but I think the code would be clearer if you made this change.

Reply:
Agreed, this is a much better solution. Typing it as `SimulationLink` requires that we use the object literal.

Fixed in 927704a